libkbfs/data_types.go

```diff
@@ -640,6 +641,28 @@ func (et EntryType) String() string {
 	return "<invalid EntryType>"
 }
 
+// EXCL indicates whether O_EXCL is set on a fuse call
+type EXCL uint32
+
+const (
+	// NoEXCL indicates O_EXCL is not set
+	NoEXCL EXCL = 0
+
+	// WithEXCL indicates O_EXCL is set
+	WithEXCL EXCL = syscall.O_EXCL
```

Comment:
It doesn't matter to me what the value is, I just don't see what advantage that gives over the standard `iota` way for constants.  It's not like we ever need to pass this value into a file system call.